aspnetcore/fundamentals/startup.md

```diff
@@ -14,82 +14,80 @@ uid: fundamentals/startup
+The `Startup` class can include a `ConfigureServices` method. This is a public method on the `Startup` class that takes an `IServiceCollection` instance as a parameter and optionally returns an `IServiceProvider`. The `ConfigureServices` method is called before `Configure`. This is necessary because some features like ASP.NET MVC require certain services to be added in `ConfigureServices` before they can be wired up to the request pipeline.
 
-Just as with `Configure`, it is recommended that features that require substantial setup within `ConfigureServices` be wrapped up in extension methods on `IServiceCollection`. You can see in this example from the default web site template that several `Add[Something]` extension methods are used to configure the app to use services from Entity Framework, Identity, and MVC:
+For features that require substantial setup there are `Add[Something]` extension methods on `IServiceCollection`. This example from the default web site template configures the app to use services for Entity Framework, Identity, and MVC:
 
 [!code-csharp[Main](../common/samples/WebApplication1/Startup.cs?highlight=4,7,11&start=40&end=55)]
 
 Adding services to the services container makes them available within your application via [dependency injection](dependency-injection.md).
 
 The `ConfigureServices` method is also where you should add configuration option classes that you would like to have available in your application. See the [Configuration](configuration.md) topic to learn more about configuring options.
 
-<a name='services-available-in-startup'></a>
-
 ## Services Available in Startup
 
-ASP.NET Core provides certain application services and objects during your application's startup. You can request certain sets of these services by simply including the appropriate interface as a parameter on your `Startup` class's constructor or one of its `Configure` or `ConfigureServices` methods. The services available to each method in the `Startup` class are described below. The framework services and objects include:
+ASP.NET Core provides certain application services and objects during an application's startup. You can request certain sets of these services by simply including the appropriate interface as a parameter on your `Startup` class's constructor or one of its `Configure` or `ConfigureServices` methods. The services available to each method in the `Startup` class are described below.
 
-**IApplicationBuilder**
+* **IApplicationBuilder**
 
-   Used to build the application request pipeline. Available only to the `Configure` method in `Startup`. Learn more about [Request Features](request-features.md).
+  Used to build the application request pipeline. Available only to the `Configure` method in `Startup`. Learn more about [Middleware](middleware.md).
```

Comment:
Do we really need ~~Available only to the `Configure` method in `Startup`.~~ 

if you really think it's needed

Available only to the `Configure` method ~~in `Startup`.~~